packages/serverless/src/apollo/app/types/collection.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line import/prefer-default-export
+export const typeCommonDefsCollection = `#graphql
+scalar JSON
+scalar Date
+
+type CollectionDescriptionOutput {
```

Comment:
Collection description should return collection schema as well, so we don't have to get schema one by one.